index.md

```diff
@@ -0,0 +1,175 @@
+| --- | --- |
+| active_period |  |
+| informed_entity | Specifiers (`agency_id`, `route_id`, etc.) in one [EntitySelector](https://gtfs.org/reference/realtime/v2/#message-entityselector) should be interpreted as being joined by the logical `AND` operator. Any combination of specifiers must exist in the GTFS data. |
+| cause |  |
+| effect |  |
+| url |  |
+| header_text |  |
+| description_text |  |
+| tts_header_text |  |
+| tts_description_text |  |
+| severity_level |  |
+### TripDescriptor
+
+| Field Name | Recommendation |
+| --- | --- |
+| trip_id | Developers need to make sure that all `trip_ids` must exist in the GTFS data, unless their `schedule_relationship` is set to `ADDED`. |
+|  | Frequency-based trips must contain `trip_id`, `start_time`, and `start_date`. |
+| route_id | Developers need to make sure that all `route_ids` must exist in the GTFS data. |
+| direction_id | Developers need to make sure that all `direction_id` must exist in the GTFS data. |
+| start_time | [Frequency-based trips](#Frequency-based-trips) must contain `trip_id`, `start_time`, and `start_date`. |
+| start_date | [Frequency-based trips](#Frequency-based-trips) must contain `trip_id`, `start_time`, and `start_date`. |
+| schedule_relationship | For [Frequency-based trips](#Frequency-based-trips), this should be `UNSCHEDULED` or empty. |
+|  | Trips that have `schedule_relationship:ADDED` must not be in GTFS data. |
+
+### VehicleDescriptor
+
+| Field Name | Recommendation |
+| --- | --- |
+| id |  |
+| label | |
+| license_plate |  |
+## Practice Recommendations Organized by Case
+
+### Frequency-based trips
+
+A frequency-based trip does not follow a fixed schedule but attempt to maintain predetermined headways. These trips are denoted in [GTFS frequency.txt](https://gtfs.org/reference/static/#frequenciestxt) by setting `exact_times=0`. There are several best practices to keep in mind when constructing GTFS-Realtime feeds for frequency-based trips.
+
+In [TripUpdate.StopTimeUpdate](#StopTimeUpdate), the [StopTimeEvent](#StopTimeEvent) for `arrival` and `departure` should not contain `delay` because frequency-based trips do not follow a fixed schedule. Instead, `time` should be provided to indicate arrival/departure predictions.
+
+Also in [TripUpdate.StopTimeUpdate](#StopTimeUpdate), `schedule_relationship` should not be `SCHEDULED` or `SKIPPED` because these values are only relevant for schedule-based trips (trips not defined in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) or in [frequencies.txt](https://gtfs.org/reference/static/#frequenciestxt) but with `exact_times=1`). As of May 2019, there is no enumeration value in`schedule_relationship` that is suitable for frequency-based trips. A new enumeration value, `UNSCHEDULED`, is being proposed to the official specification and will be the best option for frequency-based trips once approved. Until `UNSCHEDULED` is approved, we recommend leaving `schedule_relationship` empty.

```

Comment:
frequencies.txt referred and linked to two here. Is the second one supposed to be a different reference?

Reply:
I clarified the text, thanks!